lib/ansible/modules/database/postgresql/postgresql_user.py

```diff
@@ -46,16 +46,18 @@
   password:
     description:
     - Set the user's password, before 1.4 this was required.
-    - Password can be passed unhashed or hashed (MD5-hashed).
+    - Password can be passed unhashed or hashed (MD5-hashed or SCRAM-SHA256-hashed).
     - Unhashed password will automatically be hashed when saved into the
       database if C(encrypted) parameter is set, otherwise it will be save in
       plain text format.
-    - When passing a hashed password it must be generated with the format
+    - When passing an MD5-hashed password it must be generated with the format
       C('str["md5"] + md5[ password + username ]'), resulting in a total of
       35 characters. An easy way to do this is C(echo "md5$(echo -n
       'verysecretpasswordJOE' | md5sum | awk '{print $1}')").
     - Note that if the provided password string is already in MD5-hashed
       format, then it is used as-is, regardless of C(encrypted) parameter.
+    - When passing a SCRAM-hashed password it must be in internal PostgreSQL format,
+      see https://www.postgresql.org/docs/10/catalog-pg-authid.html for reference.
```

Comment:
```suggestion
      see https://www.postgresql.org/docs/current/catalog-pg-authid.html for reference.
```

Reply:
I've removed direct mentions on howto format password manually, but fixed the one in comments